Updated Layout.js

nravanelli opened the pull request:
These changes provide an option for users to select the color of the buttons (both background and border).

gfwilliams commented on modules/Layout.js:
```diff
@@ -186,8 +186,12 @@ Layout.prototype.render = function (l) {
+      btnborder = l.selected?gfx.theme.fgH:gfx.theme.fg2;
+    //add button color option
+    if(l.btnCol!==undefined) bg = l.btnCol;
+    if(l.btnBorder1!==undefined) btnborder = l.btnBorder;
```
There's a type here: `btnBorder1` - so it seems this wasn't tested?

Also, if you were to set a color, it looks like the button is then unable to show when it is selected or not (which happens on Bangle.js 1 where there is no touchscreen and you must select using buttons)? It might be better to just set the colors to the defaults if the button is selected, but then allow them to be customised when it's not selected.

... also, do we want `l.btnCol`? It might be better just to use the pre-existing `l.bgCol` value which isn't used here anyway?

nravanelli commented:
Fixed my fat finger issue, and incorporated the bgCol as suggested. When selected on BangleJS1, it _should_ default to the normal `l.selected` button color. Can't test as I have a BangleJS2.

gfwilliams commented:
Looks great - thanks!
